src/components/fabSpeedDial/fabSpeedDial.spec.js

```diff
@@ -0,0 +1,95 @@
+describe('<md-fab-speed-dial> directive', function() {
+
+  beforeEach(module('material.components.fabSpeedDial'));
+
+  function setup(template) {
+    var el;
+
+    inject(function($compile, $rootScope) {
+      el = $compile(template)($rootScope.$new());
+      $rootScope.$apply();
+    });
```

Comment:
To make testing easier, you can do something like:

```js
  pageScope = $rootScope.$new();
  element = $compile(template)(pageScope);
  scope = element.scope();
  controller = element.controller('mdFabSpeedDial');
```

This will let you make assertions against the directive's controller instance without having to make all direct DOM checks.